sky/shell/platform/ios/framework/Headers/FlutterDartProject.h

```diff
@@ -20,6 +20,8 @@ FLUTTER_EXPORT
                           packages:(NSURL*)dartPackages
     NS_DESIGNATED_INITIALIZER;
 
+- (instancetype)initDefault;
```

Comment:
Not clear what "default" means here. Maybe add a headerdoc section here or just rename. We aren't shy of using really long names in ObjC :)